pkg/driver/driver.go

```diff
@@ -864,69 +901,66 @@ func (d Driver) DeleteSnapshot(ctx context.Context, req *csi.DeleteSnapshotReque
 
 // ListSnapshots https://github.com/container-storage-interface/spec/blob/v1.2.0/spec.md#listsnapshots
 func (d Driver) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsRequest) (*csi.ListSnapshotsResponse, error) {
-	var snapshots = make([]*volume.SnapInfo, 0)
+	limit := int(req.GetMaxEntries())
+	start, err := parseStartingToken(req.GetStartingToken())
```

Comment:
"parse" is a bit common. Maybe something that transport the "convert to int"? or `var start int`. It is just not clear what that `start` could be without jumping around.

Reply:
renamed to `parseAsInt`